### DISCUSSION TITLE
Device compatibility for COVERS

### DISCUSSION THREAD

**Discussion by MacFlagg** [milo526/homebridge-tuya-web#248] [General]
Some documentation for this plug-in says COVERS are supported and other documentation says COVERS are not yet supported.

I want the TUYA plug-in to connect Quoya Smart Curtains, which are in Tuya under the MOTOR category, with HOMEKIT. Is this possible with this plug-in?

**Comment by milo526** [comment]
The cover device class **is** supported.
There is no way to check if this specific device is supported except for just trying

**Reply by MacFlagg** [reply]
Awesome and Thank You for responding so quickly.

I’ll order the Quoya rods from Amazon and give it a test. If they’re not compatible, I’ll just return them.
